docker/worker.dockerfile

```diff
@@ -1,8 +1,6 @@
 ARG FAASM_VERSION
 FROM faasm/base:${FAASM_VERSION}
 
-COPY . /usr/local/code/faasm
-
```

Comment:
We stopped doing `COPY .` a while ago as it can pull in changes specific to the local machine, and bloat the size of the image. The code is set up in the `base` image by checking out the relevant tag, so this is also not needed.